src/include/dns-server/dialog-main.rb

```diff
@@ -40,6 +40,9 @@ def initialize_dns_server_dialog_main(include_target)
       @initial_screen = "start_up"
 
       @service = SystemdService.find("named")
+
+      raise "Service 'named' not found. Please, install the required packages." unless @service
+
```

Comment:
Well, this would produce an internal error. While it's a user error.
At this state, user would be requested to report a bug, which we actually don't want, or am I wrong?